app.js

```diff
@@ -53,13 +52,58 @@ function googleAnalyticsMiddleware(data) {
     }
 }
 
+function forceUpgrade(html) {
+    var meta = [
+		"<meta http-equiv=\"Content-Security-Policy\" content=\"upgrade-insecure-requests\">"
+		].join("\n");
+    html = html.replace("</head>", meta + "\n\n</head>");
+    return html;
+}
+
+function forceHttpsUpgradeMiddleware(data) {
+    if (data.contentType == 'text/html') {
+
+        // https://nodejs.org/api/stream.html#stream_transform
+        data.stream = data.stream.pipe(new Transform({
+            decodeStrings: false,
+            transform: function(chunk, encoding, next) {
+                this.push(forceUpgrade(chunk.toString()));
+                next();
+            }
+        }));
+    }
+}
```

Comment:
Is this code used anywhere, or was it just the initial attempt and can be removed now?